projects/productconfig/.release-it.json

```diff
@@ -0,0 +1,34 @@
+{
+  "git": {
+    "requireCleanWorkingDir": true,
+    "requireUpstream": false,
+    "tagName": "productconfig-${version}",
+    "commitMessage": "Bumping productconfig version to ${version}",
+    "tagAnnotation": "Bumping productconfig version to ${version}"
+  },
+  "npm": {
+    "publishPath": "./../../dist/productconfig"
+  },
+  "hooks": {
+    "after:version:bump": "cd ../.. && yarn build:core:lib:productconfig"
```

Comment:
Make sure to invoke `build:core:lib:cds:productconfig` command here.

Reply:
@marlass what do you think, is it necessary to build all the libs here?
The same is done here: https://github.com/SAP/spartacus/blob/develop/projects/cds/.release-it.json#L13

Reply:
I think in both cases we could switch to running `ng build <lib> --prod`

Reply:
Thank you for confirming my suspicion @marlass . 
@ChristophHi  For a reference, please take a look at: https://github.com/SAP/spartacus/pull/7957/files